docs/sources/upgrade-guide/when-to-upgrade/index.md

```diff
@@ -88,22 +88,19 @@ For self-managed Grafana (both Enterprise and OSS), the support for versions is
 - Support for each minor release extends to nine months after the release date.
 - Support for the last minor release of a major version is extended an additional six months, for a total of 15 months of support after the release date.
 
-Here is an overview of projected version support through 2024:
+Here is an overview of projected version support through 2025:
 
-| **Version**               | **Release date** | **Support end of life (EOL)**          |
-| ------------------------- | ---------------- | -------------------------------------- |
-| 10.3                      | January 2024     | NO LONGER SUPPORTED as of October 2024 |
-| 10.4 (Last minor of 10.0) | March 2024       | June 2025 (extended support)           |
-| 11.0                      | May 2024         | February 2025                          |
-| 11.1                      | June 2024        | March 2025                             |
-| 11.2                      | August 2024      | May 2025                               |
-| 11.3                      | October 2024     | July 2025                              |
-| 11.4                      | December 2024    | September 2025                         |
-| 11.5                      | January 2025     | October 2025                           |
-
-{{< admonition type="note" >}}
-Grafana 9.5.x was the last supported minor for the 9.0 major release and is no longer supported as of July 2024.
-{{< /admonition >}}
+| **Version**             | **Release date** | **Support end of life (EOL)**           |
+| ----------------------- | ---------------- | --------------------------------------- |
+| 10.4 (Last minor of 10) | March 2024       | June 2025 (extended support)            |
+| 11.0                    | May 2024         | NO LONGER SUPPORTED as of February 2025 |
+| 11.1                    | June 2024        | NO LONGER SUPPORTED as of March 2025    |
```

Comment:
IMO if they're not supported we shouldn't be listing them... Can we just remove these two lines?

```suggestion
```

Reply:
I think it makes sense to start dropping the unsupported 11 minors when 10.4 is no longer supported. I think that's the least confusing way to document this vs just skipping releases. That might generate questions.

Reply:
I think @imatwawana is right on this one; I think it will maintain a level of congruity with the release cadence.